bin/build-docker.sh

```diff
@@ -53,7 +53,7 @@ EOF
     docker build "${flags[@]}" --rm=true --tag="$tag" .
     if [[ ${build_docker_post_hook:-} ]]; then
         # execute the hook, but unset it so it doesn't infinitely recurse
-        build_docker_post_hook= "$build_docker_post_hook" "$tag" "${flags[@]}" "--user=$build_run_user" --rm=true
+        build_push=$build_push build_docker_post_hook= "$build_docker_post_hook" "$tag" "${flags[@]}" "--user=$build_run_user" --rm=true
```

Comment:
Needs to be protected from undefined: build_push=${build_push:-}

Reply:
1473b0564b929fccb0a9f97bb0477625a6c53da5